pkg/kubelet/cm/dra/plugin/plugins_store.go

```diff
@@ -19,16 +19,38 @@ package plugin
 import (
 	"sync"
 
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/connectivity"
 	utilversion "k8s.io/apimachinery/pkg/util/version"
 )
 
 // Plugin is a description of a DRA Plugin, defined by an endpoint
 // and the highest DRA version supported.
 type Plugin struct {
+	sync.RWMutex
+	conn                    *grpc.ClientConn
 	endpoint                string
 	highestSupportedVersion *utilversion.Version
 }
 
+func (p *Plugin) GetGRPCConn() *grpc.ClientConn {
+	p.RLock()
+	defer p.RUnlock()
+	return p.conn
+}
+
+func (p *Plugin) SetGRPCConn(conn *grpc.ClientConn) {
+	p.Lock()
+	defer p.Unlock()
+	p.conn = conn
+}
+
+func (p *Plugin) IsConnReady() bool {
+	p.RLock()
+	defer p.RUnlock()
+	return p.conn.GetState().String() == connectivity.Ready.String()
```

Comment:
Do we really need to check string representations of the state instead of the state itself?

Reply:
done in 39d47c899f7